.github/workflows/ci.yml

```diff
@@ -140,6 +140,7 @@ jobs:
             curl -s https://raw.githubusercontent.com/Helicone/helicone/main/mitmproxy.sh | bash -s start
             agbenchmark start --maintain
           else
+            agbenchmark start --test=TestWriteFile
```

Comment:
this way we can test on pull requests, but we don't run the full testing suite to reduce costs and time